npm.sh

```diff
@@ -0,0 +1,9 @@
+set -e
```

Comment:
do you need this file? or did you want to keep this file and remove it from the circleci manifest?

Reply:
Deleted, it was an accidental commit, and _the second time_ I've commited it! :(